src/components/spreadsheet/table-wrapper.js

```diff
@@ -322,8 +343,9 @@ const TableWrapper = (props) => {
             return [];
         }
 
-        return equipments;
-    }, [equipments, props.disabled]);
+        // The equipments are formatted here too, because they are also updated by notfications in study container
+        return formatFetchedEquipments(equipments);
```

Comment:
This could cause a quite big performance loss, is redundant on initialization because data is formatted after fetching it and when rendering rows. 
It should be done only once for each needed cases, maybe add it in updateEquipments inside the reducer to cover the update through study container ?